test/libsolidity/util/TestFileParser.cpp

```diff
@@ -275,6 +275,17 @@ string TestFileParser::parseIdentifierOrTuple()
 	}
 	expect(Token::RParen);
 	identOrTuple += formatToken(Token::RParen);
+
+	while (accept(Token::LBrack))
```

Comment:
Might want to put it into a lambda - it's twice the same :-).

Reply:
Or a ``parseArrayDimensions`` function, but maybe that's overkill :-). Coming to think of it the outer function should in fact be renamed to something like ``parseSignatureArguments` as well - ``parseIdentifierOrTuple`` is not really what it does ;-).

Reply:
Yes, of course!